lib/slowio.c

```diff
@@ -82,8 +80,7 @@ EXPORTED void slowio_maybe_delay_impl(struct slowio *slowio, ssize_t n_bytes)
         return;
     }
 
-    if (slowio->last_delay.tv_sec == 0
-        && slowio->last_delay.tv_nsec == 0
+    if (slowio->last_delay.tv_sec == 0 && slowio->last_delay.tv_nsec == 0
         && slowio->bytes_since_last_delay == 0)
     {
```

Comment:
This is better than what it was doing before, but still makes it a little worse than the hand-wrapped version